mesonbuild/minstall.py

```diff
@@ -228,6 +228,9 @@ def restore_selinux_contexts() -> None:
         # is ignored quietly.
         return
 
+    if os.environ.get('DESTDIR'):
+        return
```

Comment:
nitpicking: I don't like getting it back from environ, it's nicer to do like you did for self.apply_ldconfig() and pass destdir in args, no?